src/services/waf-rules-services/list-waf-rules-tuning-attacks-service.js

```diff
@@ -2,8 +2,10 @@ import { AxiosHttpClientAdapter, parseHttpResponse } from '../axios/AxiosHttpCli
 import { makeWafRulesBaseUrl } from './make-waf-rules-base-url'
 
 export const listWafRulesTuningAttacksService = async ({ wafId, tuningId, query }) => {
+  const unconcatenateTuningIdFromIndex = tuningId.toString().slice(0, -1)
```

Comment:
Let's call it "trimmed". There is no such thing as unconcatenate
```suggestion
  const trimmedTuningId = tuningId.toString().slice(0, -1)
```